packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/gux-form-field-color.light.scss

```diff
@@ -0,0 +1,20 @@
+gux-form-field-color {
+  input {
+    &:focus,
+    &:focus-visible {
+      border: 0;
+      outline: none;
+      box-shadow: none;
+    }
+
+    &::-webkit-color-swatch-wrapper {
+      padding: 0;
+    }
+
+    &::-webkit-color-swatch,
+    &::-moz-color-swatch {
+      width: var(--gse-ui-colorPicker-input-swatchSizing);
+      height: var(--gse-ui-colorPicker-input-swatchSizing);
```

Comment:
This is good, but now that we can style the swatch, you can get a little closer to the designs by also adding:

```
  border: none;
  border-radius: var(--gse-ui-colorPicker-input-swatchBorderRadius);
```

Reply:
Once this change is in, let's ask the UX team to review it before we merge.

Reply:
@MattCheely I got the approval from the UX team.

Reply:
There's currently a conflict on style.scss, but I was thinking I could resolve this conflict when I do a git rebase when I merge this PR after it's approved.

Reply:
@MattCheely Just wanted to check in and see if the changes I pushed for this PR look ok and this PR is ready for approval. Thanks!